javascripts/imagetoscheme.html

```diff
@@ -123,18 +127,9 @@ <h1>Image to Scheme</h1>
         const output = newpixels
           .map(a =>
             a
-              .map(a => {
-                if (a < 25) return '@'
-                else if (a < 50) return '%'
-                else if (a < 75) return '#'
-                else if (a < 100) return '*'
-                else if (a < 125) return '+'
-                else if (a < 150) return '='
-                else if (a < 175) return '-'
-                else if (a < 200) return ':'
-                else if (a < 225) return '.'
-                else return ' '
-              })
+              .map(a =>
+                "@%*+=-:.  "[Math.floor(a / 25)] // what were those if-else statements
+              )
```

Comment:
```suggestion
              .map(a => '@%*+=-:.  '[Math.floor(a / 25)])
```

mostly stylistic change but also comment could just be moved to PR (it will be confusing in the file without context)